stimuli_ci_generation/src/server/gcp_cloud_storage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python
```

Comment:
so this will actually ensure it runs the first python it finds in its path, which in our case should be the right one. Source: https://stackoverflow.com/questions/2429511/why-do-people-write-usr-bin-env-python-on-the-first-line-of-a-python-script